src/plugins/condition/StyleRuleManager.js

```diff
@@ -27,22 +27,26 @@ export default class StyleRuleManager extends EventEmitter {
         super();
         this.openmct = openmct;
         this.callback = callback;
-        this.refreshData = this.refreshData.bind(this);
-        this.toggleSubscription = this.toggleSubscription.bind(this);
-        if (suppressSubscriptionOnEdit) {
-            this.openmct.editor.on('isEditing', this.toggleSubscription);
-            this.isEditing = this.openmct.editor.editing;
-        }
 
         if (styleConfiguration) {
             this.initialize(styleConfiguration);
             if (styleConfiguration.conditionSetIdentifier) {
-                this.openmct.time.on("bounds", this.refreshData);
+                this.openmct.time.on("bounds", (bounds, isTick) => {
+                    this.refreshData(bounds, isTick);
+                });
                 this.subscribeToConditionSet();
             } else {
                 this.applyStaticStyle();
             }
         }
+
+        if (suppressSubscriptionOnEdit) {
+            this.openmct.editor.on('isEditing', (isEditing) => {
+                this.toggleSubscription(isEditing);
+            });
+            this.isEditing = this.openmct.editor.editing;
```

Comment:
Should use the getter function for this `this.openmct.editor.isEditing()`. At some point someone mistakenly referenced the underlying variable, instead of using the getter, and that code has been used as a reference for a lot of subsequent code.